### PR TITLE
fix: nfsprovision claim size

### PR DESCRIPTION
**Pull request by paulopontesm**

### CODE REVIEW

File: codacy/values-production.yaml [hunk 1]
```diff
@@ -196,9 +196,12 @@ listener:
     host: <--- codacy-db-host --->
     service:
       port: 5432
+  # These configurations are only used if the `nfsserverprovisioner` is enabled
   persistence:
     claim:
-      size: 140Gi
+      # This is the size of the claim that the component will get from the provisioner (if used)
+      # Therefore this value needs to be smaller than `nfsserverprovisioner.persistence.size`
+      size: 190Gi
```
Comment (bmbferreira) [inline codacy/values-production.yaml:204]: shouldn't we do it like this? is it even possible? 😅 

`size: {{ sub Values.nfsserverprovisioner.persistence.size 10 }}`

File: codacy/values-microk8s.yaml [hunk 2]
```diff
@@ -33,9 +33,12 @@ listener:
   resources:
     limits:
       cpu: 500m
+  # These configurations are only used if the `nfsserverprovisioner` is enabled
   persistence:
     claim:
-      size: 50Gi
+      # This is the size of the claim that the component will get from the provisioner (if used)
+      # Therefore this value needs to be smaller than `nfsserverprovisioner.persistence.size`
+      size: 90Gi
```
Comment (andreaTP) [inline codacy/values-microk8s.yaml:41]: do we have those values summed up in the hw requirement documentation?

File: codacy/values-production.yaml (hunk 1 above)
Comment (andreaTP) [reply codacy/values-production.yaml:204]: I see the reasoning, having a margin of 10 Gb isn't excessive?

File: codacy/values-microk8s.yaml (hunk 2 above)
Comment (paulopontesm) [reply codacy/values-microk8s.yaml:41]: we are just asking for the 200Gi

**Pull request closed without merging**